docs/source/performance.rst

```diff
@@ -42,7 +42,7 @@ use `distributed_backend=ddp` so you can increase the `num_workers`, however you
 
 .item(), .numpy(), .cpu()
 -------------------------
-Don't call .item() anywhere on your code. Use `.detach()` instead to remove the connected graph calls. Lightning
+Don't call ``.item()`` anywhere on your code. Use ``.detach()`` instead to remove the connected graph calls. Lightning
```

Comment:
```suggestion
Don't call ``.item()`` anywhere in your code. Use ``.detach()`` instead to remove the connected graph calls. Lightning
```
This advice of not calling item ANYWHERE in your code is a bit strange. I think everyone who used PyTorch a little bit knows what .item does and when not to use it. This has nothing todo with Lightning. I think mentioning these things like .detach in this introduction is just confusing, and it has nothing todo with Lightning. 

Also, what does "Lightning takes a great deal of care to be optimized for this" mean?

Reply:
It maybe wants to say Lightning has already called .item when needed and not called when unneeded, I think?